src/main/scala/barstools/iocell/chisel/IOCell.scala

```diff
@@ -156,18 +168,25 @@ object IOCell {
     )(coreSignal:   T,
       padSignal:    T
     ): Seq[IOCell] = {
+      print("Suggested names:  " + name + "    ")
```

Comment:
Remove this print statement? Or do you think its helpful to have this here?

If you think it is helpful, we should make it more descriptive: 
"Generating IOCell [IOCellType] named [name]"

Reply:
not helpful. just missed this one. will remove